src/g_shared/a_quake.cpp

```diff
@@ -95,6 +98,22 @@ void DEarthquake::Serialize (FArchive &arc)
 	{
 		arc << m_WaveSpeedX << m_WaveSpeedY << m_WaveSpeedZ;
 	}
+	if (SaveVersion < 4525)
+	{
+		m_Falloff <<= FRACBITS;

```

Comment:
I don't even know why I bumped the save version twice. I'll fix that up, change m_Falloff to be a double and combine this and what's in SaveVersion < 4526 just below this comment.

But otherwise, yes. Randi has it set up the same way. https://github.com/rheit/zdoom/blob/156d1e61fdf2cef56442be33fd404e174a41d5a0/src/g_shared/a_quake.cpp#L71